networkml/trained_models/README.md

```diff
@@ -24,3 +24,5 @@ Currently one model is defined, depending upon the Host() featurizer. It was gen
 ~~~~
 networkml --first_stage=algorithm --final_stage=algorithm --trained_model=networkml/trained_models/host_footprint.json --label_encoder=networkml/trained_models/host_footprint_le.json --operation train --kfolds 5 ~/tmp/combined-host-ratios-041020.gz
 ~~~~
+
+Note: Our stored model uses all available features. For a full description of the statistical features used to build a model in networkML, see the README in the feaurizers directory.
```

Comment:
A couple quick things: typo: `feaurizers`, please capitalize `NetworkML` so it's consistent.

For the mention of `all available features` can you make sure that's the case? I think it's only all of the tshark `host` level features, right?

Reply:
Good catch. You're right on the host-level features. Let me add another commit that mentions that. On the tshark aspect, I think you're right. I have to admit that my language and mental model of the "features" gets a little hazy at this point. I tend to think of features as "what columns are in the dataset" (i.e. median interval arrival time of incoming packets) and I get tripped up when we talk about pyshark vs. tshark featurizers. @anarkiwi, can you please help untangle me and suggest the proper language to use here?